frontend/public/components/monitoring/receiver-forms/alert-manager-receiver-forms.tsx

```diff
@@ -358,10 +398,87 @@ const ReceiverBaseForm: React.FC<ReceiverBaseFormProps> = ({
   );
 };
 
-const ReceiverWrapper: React.FC<ReceiverFormsWrapperProps> = React.memo(({ obj, ...props }) => {
+export const SaveAsDefaultCheckbox = ({
+  formField,
+  disabled,
+  label,
+  formValues,
+  dispatchFormChange,
+  tooltip,
+}) => {
+  const saveAsDefaultLabelClass = classNames('co-nowrap', { 'co-no-bold': disabled });
   return (
+    <label className={saveAsDefaultLabelClass}>
+      <input
+        type="checkbox"
+        name={formField}
+        data-test-id="save-as-default"
+        onChange={(e) =>
+          dispatchFormChange({
+            type: 'setFormValues',
+            payload: { [formField]: e.target.checked },
+          })
+        }
+        checked={formValues[formField]}
+        disabled={disabled}
+      />
+      &nbsp; {label} &nbsp;
+      <Tooltip
+        content={
+          <div>
+            <p>{tooltip}</p>
+          </div>
+        }
+      >
+        <BlueInfoCircleIcon />
+      </Tooltip>
+    </label>
+  );
+};
+
+const ReceiverWrapper: React.FC<ReceiverFormsWrapperProps> = React.memo(({ obj, ...props }) => {
+  const { alertManagerBaseURL } = window.SERVER_FLAGS;
+  const [alertManagerGlobals, setAlertManagerGlobals] = React.useState();
+  const [errorMsg, setErrorMsg] = React.useState();
+
+  React.useEffect(() => {
+    if (!alertManagerBaseURL) {
+      setErrorMsg(`Error alertManagerBaseURL not set`);
+      return;
+    }
+    coFetchJSON(`${alertManagerBaseURL}/api/v2/status/`).then((data) => {
+      const origAlertMngrConfigStr = _.get(data, 'config.original', {});
+      if (_.isEmpty(origAlertMngrConfigStr)) {
+        setErrorMsg('alertmanager.v2.status.config.original not found.');
+      } else {
+        let origAlertMngrConfig;
+        try {
+          origAlertMngrConfig = safeLoad(origAlertMngrConfigStr);
+        } catch (e) {
+          setErrorMsg(`could not convert alertmanager config.original yaml: ${e}`);
+        }
+        setAlertManagerGlobals(origAlertMngrConfig.global);
+        setErrorMsg('');
+      }
+    });
+  }, [alertManagerBaseURL]);
+
+  if (errorMsg) {
+    return (
+      <LoadError
```

Comment:
I think if you pass `loaded` and `loadError` props to `StatusBox`, you don't need to handle these yourself

Reply:
fixed